deepchecks/core/check_utils/whole_dataset_drift_utils.py

```diff
@@ -58,6 +59,16 @@ def run_whole_dataset_drift(train_dataframe: pd.DataFrame, test_dataframe: pd.Da
                                                         random_state=random_state,
                                                         test_size=test_size)
 
+    for df in [x_train, x_test]:
+        df: pd.DataFrame
+        for column in df.columns:
+            df[column] = df[column].astype(float) if is_integer_dtype(df[column]) else df[column]
```

Comment:
Why use loop instead of pandas where? should go something like df.where(is_integer_type(df), df.astype(float), df)

Reply:
fixed no he likes it